cob_moveit_bringup/launch/stomp_moveit_planning_pipeline.launch.xml

```diff
@@ -0,0 +1,20 @@
+<launch>
```

Comment:
stick to the naming scheme and call this file `stomp_planning_pipeline.xml` in analogy to ompl